packages/server/src/open-feature.ts

```diff
@@ -104,6 +103,10 @@ export class OpenFeatureAPI extends OpenFeatureCommonAPI<Provider> implements Ma
       context
     );
   }
+
+  clearProviders() {
+    return super.clearProviders(NOOP_PROVIDER);
+  }
```

Comment:
Is it a good idea to override this method with different params?
To me this seems to break Liskovs Substitution Principle. One could not use this method in a context where a OpenFeatureCommonApi is assumed I would say.
But for some reason Typescript seems to be fine with it, maybe because it is protected. 
It still feels a bit odd, but i am not sure.

Reply:
I'll try and clean this up. I agree it's a bit odd.

Reply:
Agree this violates LSP.